### PR TITLE
Font mirror icons fix

### PR DESCRIPTION
**Pull request by fduraibi**
Updating the font file to move "spin4" and "sliders" icons to locations with no mirror property

### CODE REVIEW

File: public/fonts/fontello.html [hunk 1]
```diff
@@ -229,48 +229,48 @@
...
-      src: url('./fontello.eot?81635713');
-      src: url('./fontello.eot?81635713#iefix') format('embedded-opentype'),
-           url('./fontello.woff?81635713') format('woff'),
-           url('./fontello.ttf?81635713') format('truetype'),
-           url('./fontello.svg?81635713#fontello') format('svg');
+      src: url('./font/fontello.eot?45674533');
+      src: url('./font/fontello.eot?45674533#iefix') format('embedded-opentype'),
+           url('./font/fontello.woff?45674533') format('woff'),
+           url('./font/fontello.ttf?45674533') format('truetype'),
+           url('./font/fontello.svg?45674533#fontello') format('svg');
```
Comment (engelgabriel) [inline public/fonts/fontello.html]: need to remove the `font/ `

Comment (fduraibi) [inline public/fonts/fontello.html]: You are right, i didn't check that file I simply put it inside t he folder. But is it actually needed for the project? I feel it is just to check on the output.

**Comment by engelgabriel** [comment]
We should also fix it in a way that the following is no longger requeired:
```css
	/* Overridding the icons Unicode to suit RTL languages
	 * from _octicons.less */
	.octicon-chevron-right:before { content: '\f0a4'}
	.icon-angle-right:before { content: '\e914'; }
```

**Comment by fduraibi** [comment]
True, but that one if a different font, isn't it?

I also found the official list of all characters with mirroring property http://www.unicode.org/Public/UCD/latest/ucd/BidiMirroring.txt

**Comment by fduraibi** [comment]
I also didn't include the generated svg file thinking it was not used, and now I see that it is in the project folder under private/images. Should I include that also?

**Comment by engelgabriel** [comment]
The icon-angle-right is fontello... so we can change and fix it.
Please copy the SVG to the right place.. just in case someone need it.
Leave the octicon for now.. i'll remove it later.
Thanks!

**Comment by fduraibi** [comment]
The new commit should take care of all that.

and for the octicon, I added a new one from fontello that similar to its shape `icon-right-open-small` and its mirror `icon-left-open-small` to replace it.

**Comment by engelgabriel** [comment]
sorry.. can you review the conflicts so I can finally merge?

**Pull request merged**